helms/legion/values.yaml

```diff
@@ -239,10 +239,29 @@ feedback:
 
   output:
     # Where feedback data should be stored
-    # Only S3 is supported nowadays
     # Valid values:
     # s3 - AWS S3
-    target: s3
+    # gcs - Google Cloud Storage
+    target: "s3"
+
+    # Detail configuration for storing on GCS
+    gcs:
```

Comment:
Can we add plain authorization (by providing JSON in helm values) to allow users use our fluentd without `k8s-gke-service-account-assigner` (as we did with AWS)?

Reply:
done in e6a96cf
Add keyfile based auth  for Google Cloud:
- `keyfile` authorization option
- keyfile values should be provided in values.yaml
- chart creates configmap based on provided values and mounts it to Fluentd container